src/main/java/io/jenkins/plugins/coverage/CoveragePublisher.java

```diff
@@ -197,12 +197,12 @@ public boolean getFailBuildIfCoverageDecreasedInChangeRequest() {
     }
 
     @DataBoundSetter
-    public void setPublishChecks(boolean publishChecks) {
-        this.publishChecks = publishChecks;
+    public void setSkipPublishingChecks(boolean skipPublishingChecks) {
```

Comment:
Would it then make sense to use the same name in the warnings plugin as well? This will help users to write their pipeline scripts.